setup.py

```diff
@@ -35,6 +35,7 @@
 EXTRAS_REQUIRE = {
     "tests": TESTS_REQUIRE,
     "quality": QUALITY_REQUIRES,
+    "neuron": ["neuron-cc[tensorflow]", "torch-neuron", "protobuf==3.20.2", "torchvision"],
```

Comment:
Why not make it a base requirement?

Reply:
Some might need installing neuronx for INF2 instead of neuron for INF1.